uni/android/app/build.gradle

```diff
@@ -32,16 +32,20 @@ android {
     namespace "pt.up.fe.ni.uni"
 
     compileSdkVersion 34
-    ndkVersion flutter.ndkVersion
+    ndkVersion "25.1.8937393"
```

Comment:
Isn't this a little suspicious? Why does it happen?

Reply:
[The warning mentioned in this issue appeared](https://github.com/flutter/flutter/issues/147806).

This change isn't dangerous.

Reply:
Checked!